### ISSUE TITLE
aws_iam_role_invalid_path should allow empty string

### ISSUE THREAD

**Issue by unacceptable** [terraform-linters/tflint#1038]
## Problem statement
When writing a terraform module with a `var.path` as the IAM Role path the following error is thrown:

```console
2 issue(s) found:

Error: path must be 1 characters or higher (aws_iam_role_invalid_path)

  on iam.tf line 38:
  38:   path               = var.path

Error: "" does not match valid pattern ^(\x{002F})|(\x{002F}[\x{0021}-\x{007F}]+\x{002F})$ (aws_iam_role_invalid_path)

  on iam.tf line 38:
  38:   path               = var.path
```

I believe that the regex pattern should be something like: `(^(\x{002F})|(\x{002F}[\x{0021}-\x{007F}]+\x{002F})$|^$)` to account for modules with variables that default to empty strings (which the provider will really default to `/`).

## Version

```console
TFLint version 0.23.1
+ ruleset.aws (0.1.2-bundled)
```

**Comment by unacceptable** [comment]
I'd be happy to put in a PR for this if someone could point me in the right direction.  In the mean time I just updated my .tflint.hcl in my module like so:

``` hcl
# null path will default to "/" (Optional)
#    https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role#path
rule "aws_iam_role_invalid_path" {
  enabled = false
}
```
